### DISCUSSION TITLE
27.04.2021

### DISCUSSION THREAD

**Discussion by farag2** [farag2/Sophia-Script-for-Windows#164] [General]
# Before running the script read [Before running](https://github.com/farag2/Windows-10-Sophia-Script#fire-before-running-fire) & [How to use](https://github.com/farag2/Windows-10-Sophia-Script#how-to-use)

## Sophia Script in Action

[![YT](https://img.youtube.com/vi/f529ucAipI8/0.jpg)](https://youtu.be/f529ucAipI8)

[![YT](https://img.youtube.com/vi/MiQ85tVXQQA/0.jpg)](https://youtu.be/MiQ85tVXQQA)

***

[![ko-fi](https://www.ko-fi.com/img/githubbutton_sm.svg)](https://ko-fi.com/Q5Q51QUJC)

<a href="https://yoomoney.ru/to/4100116615568835"><img src="https://yoomoney.ru/i/shop/iomoney_logo_color_example.png" width=220px height=46px></a>

***

<a href="https://t.me/SophiaNews"><img src="https://img.shields.io/badge/Sophia%20News-Telegram-blue?style=flat&logo=Telegram"></a>
<a href="https://t.me/Sophia_Chat"><img src="https://img.shields.io/badge/Sophia%20Chat-Telegram-blue?style=flat&logo=Telegram"></a>

<img src="https://img.shields.io/github/workflow/status/farag2/Windows-10-Sophia-Script/Build?label=GitHub%20Actions&logo=GitHub">

## Windows 10 2004 (20H1)/20H2 (2009) | LTSC

Diff from v5.10.2
[5.10.2...5.10.3](https://github.com/farag2/Windows-10-Sophia-Script/compare/5.10.2...5.10.3)

* Почитайте [лонгрид](https://habr.com/company/skillfactory/blog/553800) на Хабре, как происходила разработка скрипта последние полгода.
* Closed #163;
* The `DiagnosticDataLevel` function updated;
  * You may re-run it: `DiagnosticDataLevel -Minimal` or `DiagnosticDataLevel -Default`;
* The `ErrorReporting` & `RecommendedTroubleshooting` functions updated;
* Sophia Script Wrapper updated;
  * The versiob bumped to 1.1 build 4;
  * Fixed inability to open preset file from the LTSC version.
  ![wrapper](https://i.imgur.com/joOcKi5.png)
* Minor changes. :feelsgood:
* The development focus shifted to [SophiApp](https://github.com/Sophia-Community/SophiApp) :rocket:

|              File                      |      SHA-256   |
|:--------------------------------------:|:-----------------------------:|
| Sophia.Script.LTSC.v5.2.3.zip          | 126CF01DDA127EEF0C822516E3F5B306D53BCA6CD09674FC4AA4783A8CA9D48C|
| Sophia.Script.v5.10.3.PowerShell.7.zip |D4B7A7483610CCB910ACA36480725049E05B0A9172CA5D722C878DB77E7902FA|
| Sophia.Script.v5.10.3.zip              |77B85259A91C43AF3C107D67CCC8A6B719B1A8F43F73F0E4C7C017B7BF9AEE15|
| Sophia.Script.Wrapper.zip              |C2B03ABCA29B29D273C3D92EAD30168FAA90E851B6CB685BA7E1AEE5D6AAF07C|

Learn more about David's [wrapper](https://github.com/farag2/Windows-10-Sophia-Script/tree/master/Wrapper) from [benchtweakgaming.com](https://benchtweakgaming.com/2020/11/12/windows-10-debloat-tool/).

[![YT](https://img.youtube.com/vi/f529ucAipI8/1.jpg)](https://www.youtube.com/watch?v=f529ucAipI8&t=946s)

<hr /><em>This discussion was created from the release <a href='https://github.com/farag2/Windows-10-Sophia-Script/releases/tag/5.10.3'>27.04.2021</a>.</em>

**Comment by ghost** [comment]
Here I come again... Still about the sum file. It turns out that I was wrong and a typical `SHA256SUM` should be like (note there are *two* spaces between hash and filename, though it doesn't matter in most cases):

    126CF01DDA127EEF0C822516E3F5B306D53BCA6CD09674FC4AA4783A8CA9D48C  Sophia.Script.LTSC.v5.2.3.zip
    D4B7A7483610CCB910ACA36480725049E05B0A9172CA5D722C878DB77E7902FA  Sophia.Script.v5.10.3.PowerShell.7.zip
    77B85259A91C43AF3C107D67CCC8A6B719B1A8F43F73F0E4C7C017B7BF9AEE15  Sophia.Script.v5.10.3.zip
    C2B03ABCA29B29D273C3D92EAD30168FAA90E851B6CB685BA7E1AEE5D6AAF07C  Sophia.Script.Wrapper.zip

Sorry for the inconvenience.

**Reply by farag2** [reply]
I'll re-upload it tomorrow. Think I should automate this process too)
